src/setupTests.js

```diff
@@ -1,33 +1,23 @@
-import React from 'react'
-import { act } from 'react-dom/test-utils'
 import Adapter from 'enzyme-adapter-react-16'
-import { configure, mount } from 'enzyme'
+import { configure } from 'enzyme'
 import 'jest-localstorage-mock'
+import './store'
```

Comment:
In some reducers test store wasn't getting initialized before store is subscribed or action is dispatched causing issues. So I imported store here to make sure it is initialized before every test file. I will make sure if it needed or not after the changes I have made. If not I will remove this.

Reply:
Most tests do not need the global `store` since they initialize state themselves and call reducers directly. Only some mount tests and some shortcut tests should require the global store. Even those should likely be refactored to accept a mock store via dependency injection.

Reply:
> Most tests do not need the global `store` since they initialize state themselves and call reducers directly. Only some mount tests and some shortcut tests should require the global store. Even those should likely be refactored to accept a mock store via dependency injection.

That makes sense. We should use dependency injection to mock store too.